apps/omg_eth/test/fixtures.exs

```diff
@@ -56,12 +56,16 @@ defmodule OMG.Eth.Fixtures do
     {:ok, true} =
       Ethereumex.HttpClient.request("personal_unlockAccount", ["0x6de4b3b9c28e9c3e84c2b2d3a875c947a84de68d", "", 0], [])
 
-    add_exit_queue =
-      RootChainHelper.add_exit_queue(1, @eth, %{
-        plasma_framework: Encoding.from_hex("0xc673e4ffcb8464faff908a6804fe0e635af0ea2f")
-      })
+    case RootChainHelper.has_exit_queue(1, @eth, contract.contract_addr) do
+      {:ok, false} ->
+        add_exit_queue = RootChainHelper.add_exit_queue(1, @eth, contract.contract_addr)
+
+        {:ok, _} = Support.DevHelper.transact_sync!(add_exit_queue)
+
+      _ ->
```

Comment:
1. pattern match on `{:ok, true}` for an early fail if the request borks
2. it is a bit strange that we didn't require the `case` earlier and now we do. I'm guessing there's a reason why sometimes the queue is there and sometimes it isn't? Though it is strange, since we're working off of the same snapshot every time.